python_ta/debug/AccumulationTable.py

```diff
@@ -0,0 +1,72 @@
+import inspect
+
+import astroid
+import tabulate
+
+
+class AccumulationTable:
+    _num_iterations: int
+    _accumulation_names: list
+    _loop_accumulation_values: dict
+    _loop_var_name: str
+    _loop_variable: list
+
+    def __init__(self, accumulation_names: list):
+        self._accumulation_names = accumulation_names
+        self._num_iterations = 0
+        self._loop_variable = ["N/A"]
```

Comment:
it's okay if there's no loop variables specified (you can make this an empty list)

Reply:
For this specific thing it throws off the formatting of the table: it makes it so the loop variable section is now missing the zeroth iteration, so either I can leave it as it is or I can use '\n' instead